fix(apple silicon): fixed various typos and product names (#6694)
scaleway/docs-content: pages/apple-silicon/concepts.mdx

Apple silicon as-a-Service is tailored for developing, building, testing, and signing applications for Apple devices such as iPhones, iPads, Mac computers, and more. The Mac mini boasts a sophisticated neural engine that significantly enhances machine learning capabilities.

## Asahi Linux
Asahi Linux is a Linux distribution designed for Apple silicon Mac minis. It enables running a native Linux kernel on the ARM64 architecture, with low-level hardware support for Mac components. The M2 Pro Mac minis configured for Asahi Linux are dedicated to running this operating system and are accessible remotely via SSH.
The integrated GPU provides accelerated support for graphical environments and 3D applications, thanks to actively developed open-source drivers. However, there is currently no open-source driver available for AI hardware acceleration (NPU). Asahi Linux brings full [Fedora Linux](https://www.fedoraproject.org/) experience to Mac hardware.

<Message type="note">

>   Asahi Linux is currently in experimental status and we can not guarantee seamless user experience using this distribution on a Mac mini.

Reviewer: ```suggestion
  Asahi Linux is currently in experimental status and we cannot guarantee seamless user experience using this distribution on a Mac mini.
```
